app/controllers/application_controller/ci_processing.rb

```diff
@@ -90,12 +90,12 @@ def build_ownership_info
     Rbac.filtered(User).each { |u| @users[u.name] = u.id.to_s }
     record = klass.find(@ownership_items[0])
     user = record.evm_owner if @ownership_items.length == 1
-    @user = user ? user.id.to_s : nil
+    @user = user.try(:id).to_s
```

Comment:
This isn't the same. `@user` will be an empty string instead of `nil` if `user` is falsey.

It may not matter, but as the goal of this PR is just a refactor you might want to change this to `@user = user.id if user`  (and the other two references in this file)